src/SourceBuild/patches/nuget-client/0002-Add-dependencies-for-PVP-flow.patch

```diff
@@ -0,0 +1,60 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Nikola Milosavljevic <nikolam@microsoft.com>
+Date: Wed, 7 Jun 2023 03:03:22 +0000
+Subject: [PATCH] Add dependencies for PVP flow
+
+Backport: https://github.com/NuGet/NuGet.Client/pull/5207
+---
+ Directory.Packages.props |  1 +
+ eng/Version.Details.xml  | 27 +++++++++++++++++++++++++++
+ 2 files changed, 28 insertions(+)
+
+diff --git a/Directory.Packages.props b/Directory.Packages.props
+index baec21914..2461e41f0 100644
+--- a/Directory.Packages.props
++++ b/Directory.Packages.props
+@@ -16,6 +16,7 @@
+     <MicrosoftBuildVersion Condition="'$(TargetFramework)' == 'netcoreapp5.0'">16.11.0</MicrosoftBuildVersion>
+ 
+     <!-- Overridden by source build to ensure the same version is used across products, do not remove these properties -->
++    <!-- For each property here, there is an appropriate package dependency in eng/Version.Details.xml -->
```

Comment:
In order to reduce the chance of patch conflicts, this change should be removed since it provides no functional behavior.

Reply:
Yeah, makes sense. I'll remove it from the patch.

Reply:
Fixed with https://github.com/dotnet/installer/pull/16603/commits/4604dec8ae5ea67884908005f1254680316ed6f4